orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/rollback/ExplicitRollback.groovy

```diff
@@ -92,12 +92,15 @@ class ExplicitRollback implements Rollback {
       ]
     }
 
-    def stages = [enableServerGroupStage]
+    // https://github.com/spinnaker/spinnaker/issues/4895 - capture the source capacity as early as possible
+    def stages = []
     if (!parentStage.getContext().containsKey("sourceServerGroupCapacitySnapshot")) {
       // capacity has been previously captured (likely as part of a failed deploy), no need to do again!
```

Comment:
this comment confuses me

Reply:
me too ... otherwise looks good

Reply:
I mean... it comes from 

```commit ed90df775b9e5870c2c6779818293c90a34edb7d (tag: v6.4.0)
Author: Adam Jordens <adam@jordens.org>
Date:   Tue Feb 13 10:40:12 2018 -0800

    feat(rollback): Support for automatic rollback when a deployment fails (#1944)
    
    This PR injects a `RollbackCluster` stage if a particular cluster
    deployment has automatic rollback enabled.
```

:)

Reply:
My comment about the comment was more that it doesn’t really describe much, I find little value in linking out to issues that are also linked out to in the commit message itself. Perhaps others do tho.

Reply:
I see, we're not talking about the same thing. I thought @marchello2000's comment was referring to this:

`// capacity has been previously captured (likely as part of a failed deploy), no need to do again!`

For me linking out from the commit and from the comment serve different purposes: linking from the commit makes it visible to github on the issue, and linking in the comment makes it readable from the source without having to dig through potentially layers of git blame.